aws_lambda_powertools/utilities/feature_flags/schema.py

```diff
@@ -325,9 +325,9 @@ def validate_condition_key(condition: Dict[str, Any], rule_name: str):
 
     @staticmethod
     def validate_condition_value(condition: Dict[str, Any], rule_name: str):
-        value = condition.get(CONDITION_VALUE, "")
-        if not value:
-            raise SchemaValidationError(f"'value' key must not be empty, rule={rule_name}")
+        value = condition.get(CONDITION_VALUE)
+        if value is None:
+            raise SchemaValidationError(f"'value' key must not be none, rule={rule_name}")
```

Comment:
```suggestion
            raise SchemaValidationError(f"'value' key must not be null, rule={rule_name}")
````

`null` is more accurate and transferrable to other languages in the future.